### PR TITLE
Progress bar color

### PR DESCRIPTION
**Pull request by kisslorand**
### Description

Color choices added for progress bar during print.
The choices are: 
 - Orange
 - Yellow
 - Red
 - Green
 - Blue
 - Cyan
 - Magenta
 - Purple
 - Lime
 - Gray

The color of the progress bar can be set at compile time in the "Configuration.h" by setting `PROGRESS_BAR_COLOR`.

### Related Issues

#2272

### CODE REVIEW

**Comment by xPew** [comment]
@kisslorand Thank you for this PR. It changes the color of the progress bar indeed. A config.ini option would be really nice, but it all comes down to memory :/
Also changes this line what radek suggested and it solves my other problem. Do you know a more permanent fix for this one by any chance? Maybe it consumes less memory?
image
I am willing to listen to the programmers, I just ask these for the basic users :D

File: TFT/src/User/Configuration.h [hunk 1]
```diff
@@ -148,6 +148,13 @@
  */
 #define TERMINAL_COLOR_SCHEME 0  // Default: 0
 
+/**
+ * Progress bar color
+ * The color of the progress bar during print
+ *   Options: [Orange: 0, Yellow: 1, Red: 2, Green: 3, Blue: 4, Cyan: 5, Magenta: 6, Purple: 7, Lime: 8, Gray: 9]
+ */
+#define PROGRESS_BAR_COLOR 0  // Default: 0
+
```
Comment (digant73) [inline TFT/src/User/Configuration.h:157]: you should move that setting after TERMINAL_KEYBOARD_LAYOUT (so in the param configurable at compile time).
Also please restore //#define MARKED_PROGRESS_BAR  // Default: commented (disabled). IOn this case // is not used for a comment but to disable a macro (or in general an instruction)

File: TFT/src/User/SanityCheck.h [hunk 2]
```diff
@@ -51,6 +51,18 @@ extern "C" {
...
+    #error "PROGRESS_BAR_COLOR cannot be greater than 9"
+  #endif
+
+  #if PROGRESS_BAR_COLOR < 0
+    #error "PROGRESS_BAR_COLOR cannot be less than 0"
+  #endif
+#else
+  #define PROGRESS_BAR_COLOR 0
+#endif
+
```
Comment (digant73) [inline TFT/src/User/SanityCheck.h:65]: same as in Configuration.h. It should be moved after TERMINAL_KEYBOARD_LAYOUT

**Review by digant73 — CHANGES_REQUESTED** [review]

File: TFT/src/User/Configuration.h (hunk 1 above)
Comment (kisslorand) [reply TFT/src/User/Configuration.h]: Done.

File: TFT/src/User/SanityCheck.h (hunk 2 above)
Comment (kisslorand) [reply TFT/src/User/SanityCheck.h]: Done.

**Pull request merged**